fades/main.py

```diff
@@ -63,6 +63,32 @@ def _merge_deps(*deps):
     return final
 
 
+def detect_inside_virtualenv():
+    """Tell if fades is inside a virtualenv.
+
+    This is a slightly adapted version from ipython's own check.
+    """
+    p_venv = os.environ.get('VIRTUAL_ENV')
```

Comment:
I think that the best way to check if is running inside a venv is with sys.prefix. With the environment variable we are not covering all the cases. 

More: https://www.python.org/dev/peps/pep-0405/#specification
Example: https://github.com/pypa/pip/blob/281eb61b09d87765d7c2b92f6982b3fe76ccb0af/pip/locations.py#L39